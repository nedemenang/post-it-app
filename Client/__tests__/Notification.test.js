import React from 'react';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';
import NotificationItem from '../components/notification.jsx';

describe('the notificaiton component', () => {
  let mountedComponent;

  const NotificationComponent = () => {
    if (!mountedComponent) {
      mountedComponent = shallow(<NotificationItem />);
    }
    return mountedComponent;
  };

  beforeEach(() => {
    mountedComponent = undefined;
  });

  it('should render', () => {
    const component = NotificationComponent();
    expect(component).toBeDefined();
  });

  it('should match snapshot test', () => {
    const component = NotificationComponent();
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
});
