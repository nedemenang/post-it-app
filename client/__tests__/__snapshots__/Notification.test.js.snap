// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the notificaiton component should match snapshot test 1`] = `
<div>
  <Snackbar
    autoHideDuration={4000}
    message="Message added in group "
    onRequestClose={[Function]}
    open={false}
  />
</div>
`;
